tuxsuite/next-clang-13.tux.yml

```diff
@@ -2,7 +2,7 @@
 # This file has been autogenerated by invoking:
 # $ ./generate_tuxsuite.py next
 # Invoke tuxsuite via:
-# $ tuxsuite plan --git-repo https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git --git-ref master --job-name defconfigs --json-out builds.json  --patch-series patches/nexttuxsuite/next-clang-13.tux.yml
```

Comment:
I think we have a bug in the generator...

Shouldn't there be whitespace between `patches/next` and `tuxsuite/...`?

Reply:
https://github.com/ClangBuiltLinux/continuous-integration2/issues/431